sdk/search/azure-search/src/main/java/com/azure/search/SearchIndexClientBuilder.java

```diff
@@ -121,140 +170,139 @@ public SearchIndexClientBuilder endpoint(String endpoint) {
     }
 
     /**
-     * Sets the index name
+     * Sets the {@link SearchApiKeyCredential} used to authenticate HTTP requests.
      *
-     * @param indexName name of the index
-     * @return the updated SearchIndexClientBuilder object
+     * @param searchApiKeyCredential SearchApiKeyCredential used to authenticate HTTP requests.
```

Comment:
Add @link for SearchApiKeyCredential . The SearchApiKeyCredential `is` used to authenticate HTTP requests.